wafer_space/projects/url_handlers.py

```
@@ -144,6 +145,95 @@ def post_download(self, content: bytes, metadata: dict[str, Any]) -> bytes:
+        return {
+            "url": url,  # Keep original URL, download task will use API
+            "metadata": {
+                "handler": "GitHubArtifactHandler",
+                "owner": match.group("owner"),
+                "repo": match.group("repo"),
+                "run_id": match.group("run_id"),
+                "requires_github_auth": True,
+            },
+        }
+
+    def post_download(self, content: bytes, metadata: dict[str, Any]) -> bytes:
```

Ruff (0.14.5)

Unused method argument: `metadata`

(ARG002)